modules/booklist.js

```diff
@@ -0,0 +1,81 @@
+export default class BookList {
+  constructor() {
+    this.form = document.querySelector('#book-form');
+    this.bookListContainer = document.querySelector('#book-list');
+    this.books = [];
+    this.empty = {};
+    this.loadBooksFromLocalStorage();
+    this.displayBooks();
+  }
+
+  book(title, author) {
+    let obj = this.empty;
+    obj = {
+      title,
+      author,
+    };
+    return obj;
+  }
+
+  loadBooksFromLocalStorage() {
+    const storedBooks = localStorage.getItem('books');
+    if (storedBooks) {
+      this.books = JSON.parse(storedBooks);
+    } else {
+      localStorage.setItem('books', JSON.stringify(this.books));
+    }
+  }
+
+  displayBooks() {
+    this.bookListContainer.innerHTML = '';
+    this.books.forEach((book, index) => {
+      const li = document.createElement('div');
+      function capitalize(name) {
```

Comment:
* As stated in the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Refactor%20your%20functions%20to%20arrow%20functions.), Kindly Refactor all your functions to arrow functions.